packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -157,11 +166,50 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
     );
   }
 
+  private _onLabelWrapperRef(element: HTMLDivElement | null): void {
+    if (element) {
+      const labelWrapperElement = createRef<HTMLDivElement>();
```

Comment:
hmm. usually createRef isn't done here. confused.

Reply:
Originally I was having an issue with the tooltip getting confused about which element had overflow, but I tried again and now it's working without the extra createref!